mmv1/products/redis/api.yaml

```diff
@@ -394,3 +394,57 @@ objects:
           - :READ_REPLICAS_DISABLED
           - :READ_REPLICAS_ENABLED
         input: true
+      - !ruby/object:Api::Type::NestedObject
+        name: persistenceConfig
+        description: Configuration of the persistence functionality.
+        min_version: beta
+        allow_empty_object: true
+        send_empty_value: true
+        properties:
+        - !ruby/object:Api::Type::Enum
+          name: persistenceMode
+          min_version: beta
+          description: |
+            Optional. 
+            Controls whether Persistence features are enabled. 
+            If not provided, the existing value will be used.
+            - PERSISTENCE_MODE_UNSPECIFIED: Not set.
+            - DISABLED: Persistence is disabled for the instance, and any existing snapshots are deleted.
+            - RDB: RDB based Persistence is enabled.
+          values:
+            - :PERSISTENCE_MODE_UNSPECIFIED
+            - :DISABLED
+            - :RDB
+          input: true
+        - !ruby/object:Api::Type::Enum
+          min_version: beta
+          name: rdbSnapshotPeriod
+          description: |
+            Optional. 
+            Period between RDB snapshots. Snapshots will be attempted every period starting from the provided snapshot start time.
+            - SNAPSHOT_PERIOD_UNSPECIFIED: Not set.
```

Comment:
Same deal w/ UNSPECIFIED.

Reply:
Thanks for the comments @rileykarson, I've updated the PR accordingly.

If you don't mind me asking, are there plans in the future to grant log viewing rights specifically to the author of PRs (even for non google contributors) in the future?  This might help offload the debugging from your workloads